consensus/hotstuff/committee.go

```diff
@@ -49,6 +49,15 @@ type Committee interface {
 	DKG(blockID flow.Identifier) (DKG, error)
 }
 
+type BlockSignerDecoder interface {
```

Comment:
an implementation of this API can be injected into the various APIs, to enable decoding of signer indices